src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -0,0 +1,167 @@
+        salesForceContacts.clickContactNameLink();
+        salesForceContacts.clickEditContact();
+
+    }
+    /**
+     * And step.
+     */
+    @And("^I edit the contact name field and I press the save button")
+    public void IEditTheContactNameFieldAndIPressTheSaveButton() {
+        System.out.println("This Step edits contact name field");
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceContacts = new SalesForceContacts(webDriver);
+        salesForceContacts.editContact(propertiesManager.getProperty("contactEditName"));
+        salesForceContacts.clickSaveContactButton(salesForceContacts.getSaveContactButton());
+
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^The system shows the modified contact$")
+    public void TheSystemShowsTheModifiedContact() {
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceContacts = new SalesForceContacts(webDriver);
+        System.out.println("This Step tests modified contact:" + salesForceContacts.newContactSavedName());
+        StringJoiner name = new StringJoiner(" ");
+        name.add(propertiesManager.getProperty("contactEditName"));
+        name.add(propertiesManager.getProperty("contactLastName"));
+        Assert.assertEquals(salesForceContacts.newContactSavedName(), name.toString());
+    }
+    /**
+     * When step.
+     */
+    @When("^I choose a contact from recent contacts and I click on delete contact")
+    public void IChooseAContactFromRecentContactsAndIClickOnDeleteContact() {
+        System.out.println("Choose last contact");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceContacts = new SalesForceContacts(webDriver);
+        salesForceContacts.clickContactNameLink();
+        salesForceContacts.clickDeleteContact();
+    }
+    /**
+     * And step.
+     */
+    @And("^I click on ok button")
+    public void IClickOnOkButton() {
+        System.out.println("Delete contact");
+        webDriver = DriverManager.getInstance().getNavigator();
+        salesForceContacts = new SalesForceContacts(webDriver);
+        salesForceContacts.clickDeleteAlert();
+
+    }
+    /**
+     * Then step.
+     */
+    @Then("^The system deletes the selected contact$")
+    public void TheSystemDeletesTheSelectedContact() {
+        webDriver = DriverManager.getInstance().getNavigator();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        salesForceContacts = new SalesForceContacts(webDriver);
+        System.out.println("This Step tests verified id the contact:");
+        Assert.assertEquals(salesForceContacts.contactHomePage(), "Home");
+    }
+
+
+    /**
+     * After, to close browser after the tests are executed.
+     */
+    /*@After
+    public void closeBrowser() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)